### PR TITLE
My Javascript Game

### PR DESCRIPTION
**Pull request by andrewbaldwin44**

### CODE REVIEW

**Comment by andrewbaldwin44** [comment]
https://andrewbaldwin44.github.io/project-m2-nyan-cat/

File: lib/js/serialization.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function serialize(score) {
```
Comment (scottanthonymorin) [inline lib/js/serialization.js:1]: Nice

File: lib/js/main.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  document.removeEventListener('keydown', keydownHandler);
+}
+
+const gameApp = document.getElementById('app');
+const gameEngine = new Engine(gameApp);
+
+const movementKeys = ['ArrowLeft', 'ArrowRight', 'ArrowUp', 'ArrowDown'];
+const keysPressed = {};
+
+let bacon = 0;
```
Comment (scottanthonymorin) [inline lib/js/main.js:46]: Finally! A legitimate use of the word `bacon` in a repo. lol

File: lib/js/engine-utilities.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  enemy_second_image = ENEMY_SECOND_IMAGE;
+  enemy_third_image = ENEMY_THIRD_IMAGE;
+  enemy_bullet_image = ENEMY_BULLET_IMAGE;
+  enemy_boss_image = ENEMY_BOSS_IMAGE;
+
+  bg.src = background_image;
+  gameEngine.updatePlayerImage();
+}
+
+function setScott() {
```
Comment (scottanthonymorin) [inline lib/js/engine-utilities.js:109]: LOL

File: lib/js/data.js [hunk 4]
```diff
@@ -0,0 +1,137 @@
+const GAME_WIDTH = 450;
```
Comment (scottanthonymorin) [inline lib/js/data.js:1]: This file is impressive

**Pull request closed without merging**